src/java/main/org/apache/zookeeper/common/X509Util.java

```diff
@@ -150,7 +196,7 @@ public static X509KeyManager createKeyManager(String keyStoreLocation, String ke
             KeyStore ks = KeyStore.getInstance("JKS");
             inputStream = new FileInputStream(keyStoreFile);
             ks.load(inputStream, keyStorePasswordChars);
-            KeyManagerFactory kmf = KeyManagerFactory.getInstance("SunX509");
+            KeyManagerFactory kmf = KeyManagerFactory.getInstance("PKIX");
```

Comment:
A comment has been added where pkix is used for the trustmanager factory. We don't actually need this change for the keymanager, but i preferred to be consistent.